packages/frontend/.parcelrc

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "@parcel/config-default",
+  "validators": {
+    "*.{ts,tsx}": ["@parcel/validator-typescript"]
```

Comment:
Is this also doing type checking ? I see you had to ignore typescript on one of the component files